win2xcur/multiscale.py

```diff
@@ -0,0 +1,32 @@
+from win2xcur.cursor import CursorFrame, CursorImage
+
+MULTSCALE = [16, 24, 32, 48, 64, 96, 128, 192, 256]
+
+def generates_frames(cursor, min_size: int) -> None:
+    """Generates multiple sizes for each cursor.
+
+    Args:
+        cursor (Cursor): The cursor to generate sizes for.
+        min_size (int): The minimum size to generate.
+
+    Returns:
+        List[Cursor]: The generated cursors.
```

Comment:
This directly contradicts the `-> None`. In general, I don't like declaring signatures in comments when you can explicitly declare them in the signature. DRY.